utils/aws_api.py

```diff
@@ -570,6 +570,25 @@ def init_ecr_auth_tokens(self, accounts):
         self.auth_tokens = auth_tokens
 
     def get_cluster_vpc_id(self, account):
+        """
+        Returns a cluster VPC ID.
+
+        :param account: a dictionary containing the following keys:
+                        - name - name of the AWS account
+                        - assume_role - role to assume to get access
+                                        to the cluster's AWS account
+                        - assume_region - region in which to operate
+                        - assume_cidr - CIDR block of the cluster to
+                                        use to find the matching VPC
+        """
+        required_keys = \
+            ['name', 'assume_role', 'assume_region', 'assume_cidr']
+        ok = all(elem in account.keys() for elem in required_keys)
+        if not ok:
+            account_name = account.get('name')
```

Comment:
nit: being strict, you may not have `name` here 😅

Reply:
i know, but it's the best we can do...

Reply:
What do you mean by "the best we can do"? If you cannot be sure that you won't have `name` in the input dict, you cannot place it there (unless add more conditions)

Reply:
well, `name` is a mandatory field in the schema. it will be here for sure.